test/server/cards/01_SOR/units/JabbaTheHuttCunningDaimyo.spec.ts

```diff
@@ -0,0 +1,41 @@
+describe('Jabba the Hutt, Cunning Daimyo', function () {
+    integration(function (contextRef) {
+        describe('Jabba the Hutt\'s ability', function () {
```

Comment:
Can we also test that it does not reduce non-Trick events, and that it does not reduce Trick events played by the opponent?